bcipy/signal/model/mach_learning/dimensionality_reduction.py

```diff
@@ -0,0 +1,95 @@
+import numpy as np
```

Comment:
File is actually just moved from `bcipy/signal/model/mach_learning/dimensionality_reduction/function_dim_reduction.py`